test/lib/testAttributeEditorTemplate.py

```diff
@@ -66,6 +65,68 @@ def searchForMayaControl(self, controlOrLayout, mayaControlCmd, labelToFind):
                         if mayaControlCmd(child, q=True, label=True) == labelToFind:
                             return child
         return None
+    
+    def findAllFrameLayoutLabels(self, startLayout):
+        '''
+        Find all child frame layout and return a list of their labels.
+        '''
+        found = []
+        
+        if not startLayout:
+            return found
+            
+        childrenOfLayout = cmds.layout(startLayout, q=True, ca=True)
+        if not childrenOfLayout:
+            return found
+            
+        for child in childrenOfLayout:
+            child = startLayout + '|' + child
+            if cmds.frameLayout(child, exists=True):
+                found.append(cmds.frameLayout(child, q=True, label=True))
+            elif cmds.layout(child, exists=True):
+                found += self.findAllFrameLayoutLabels(child)
+                    
+        return found
+
+    def findExpandedFrameLayout(self, startLayout, labelToFind, primPath):
```

Comment:
Now that sections are mosyly non-expanded, existing test needed to be modified to expand the sections, otherwise they would not find their expected UI.